### ISSUE TITLE
skillset missing return

### ISSUE THREAD

**Issue by Rescator7** [tbamud/tbamud#80]
modify.c
in ACMD(do_skillset)

} else if (spell_info[skill].min_level[(pc)] > pl) {
    send_to_char(ch, "%s is a level %d %s.\r\n", GET_NAME(vict), pl, pc_class_types[pc]);
    send_to_char(ch, "The minimum level for %s is %d for %ss.\r\n", spell_info[skill].name, spell_info[skill].min_level[(pc)], pc_class_types[pc]);
  }

return is missing at the end of the score. it continue and set the skill, even if it said it shouldn't.

} else if (spell_info[skill].min_level[(pc)] > pl) {
    send_to_char(ch, "%s is a level %d %s.\r\n", GET_NAME(vict), pl, pc_class_types[pc]);
    send_to_char(ch, "The minimum level for %s is %d for %ss.\r\n", spell_info[skill].name, spell_info[skill].min_level[(pc)], pc_class_types[pc]);
    return;
  }

**Comment by welcor** [comment]
I don't think this is as much a bug as a feature - this allows a god to set a skill even though the requirements aren't met.
It just lets you know.

Specifically this is used by immorts for testing - setting their own skill low, testing stuff and then setting it high again, regardless of previous class..

**Comment by Rescator7** [comment]
Ah i see. it's fine. I thought, it could be that, but it wasn't clear. Because, if it's for testing, or IMM power. I thought why allow to set skills out of LEVEL, but not also to OUT of CLASS? which isn't allowed. 

BTW, the message is pretty weird to setskill out of class: %s cannot be learned by mortals.
Actually, it can be learned by mortal, but not sometimes in a  particuliar class.

500H 100M 82V (news) (motd) > skillset bibi 'bash' 100
bash cannot be learned by mortals.

bibi is class mu, and we know bash is for mortal warrior

Bob

**Comment by welcor** [comment]
Suggestion: better warning message when assigning skills via skillset

**Comment by Rescator7** [comment]
Yes, that would be great. The current one is miss leading.

**Comment by riotcore** [comment]
Ah, my mistake, I missed the earlier discussion that this behavior was intentional. Sorry about that!
